scripts/eosio_build_darwin.sh

```diff
@@ -209,32 +212,31 @@
 			done
 		fi
 		printf "\tInstalling boost libraries.\n"
-		cd ${TEMP_DIR}
-		STATUS=$(curl -LO -w '%{http_code}' --connect-timeout 30 https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.bz2)
-		if [ "${STATUS}" -ne 200 ]; then
-			printf "\tUnable to download Boost libraries at this time.\n"
-			printf "\tExiting now.\n\n"
-			exit;
-		fi
-		tar xf ${TEMP_DIR}/boost_1_66_0.tar.bz2
-		rm -f  ${TEMP_DIR}/boost_1_66_0.tar.bz2
-		cd ${TEMP_DIR}/boost_1_66_0/
-		./bootstrap.sh "--prefix=$BOOST_ROOT"
-		if [ $? -ne 0 ]; then
-			printf "\n\tInstallation of boost libraries failed. 0\n"
-			printf "\n\tExiting now.\n"
-			exit 1
-		fi
-		./b2 install
-		if [ $? -ne 0 ]; then
-			printf "\n\tInstallation of boost libraries failed. 1\n"
-			printf "\n\tExiting now.\n"
-			exit 1
+		if [ ! -d "/usr/local/Cellar/boost/1.66.0" ]; then
+			if ! brew install "https://raw.githubusercontent.com/Homebrew/homebrew-core/76a83d798d76aa43459ceb145b721ba4a16a54a6/Formula/boost.rb"
+			then
+				printf "\tUnable to install boost 1.66 libraries at this time. 0\n"
+				printf "\tExiting now. 0\n\n"
```

Comment:
Are these debug numbers being left in deliberately?

Reply:
Yes, but good catch. Only the first printf should have the 0. It's not an absolute requirement that  these are left in there, but it does save time when users open issues.